suite/auto-sync/README.md

```diff
@@ -15,6 +15,9 @@ Please refer to [intro.md](intro.md) for an introduction about this tool.
 
 ## Install
 
+> ⚠️
```

Comment:
Please use the Github markdown highlighting for this:
https://github.com/orgs/community/discussions/16925